warp10/src/main/java/io/warp10/hadoop/Warp10InputFormat.java

```diff
@@ -161,6 +161,7 @@ public class Warp10InputFormat extends InputFormat<Text, BytesWritable> {
   public static final String PROPERTY_WARP10_FETCH_TIMESTEP = "warp10.fetch.timestep";
   public static final String PROPERTY_WARP10_FETCH_PREBOUNDARY = "warp10.fetch.preboundary";
   public static final String PROPERTY_WARP10_FETCH_POSTBOUNDARY = "warp10.fetch.postboundary";
+  public static final String PROPERTY_WARP10_FETCH_ATTR_PREFIX = "warp10.fetch.attr.";
```

Comment:
Wouldn't it be better to compose the name with warp10.fetch. + HTTP_PARAM_ATTR_PREFIX ?

Reply:
not supposed to change, and in case of change better not to propagate it.